mappings/net/minecraft/datafixers/schemas/Schema2100.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_4465 net/minecraft/datafixers/schemas/Schema2100
```

Comment:
This is very crucial; my game currently crashes in dev env for Schema100 needs to be in the same package (`net.minecraft.datafixers.schemas`) as this class